src/test/resources/logback.xml

```diff
@@ -9,16 +9,29 @@
     	</appender>
     <timestamp key="date" datePattern="yyyyMMddHHmmss"/>
 
+    <!-- uncomment and set to DEBUG to log all failing HTTP requests -->
+    <!-- uncomment and set to TRACE to log all HTTP requests -->
+    <!-- <logger name="io.gatling.http.engine.response" level="TRACE" />-->
+
     <appender name="FILE" class="ch.qos.logback.core.FileAppender">
         <file>target/simulation-${date}.log</file>
         <encoder>
             <pattern>%d{HH:mm:ss.SSS} [%-5level] %logger{15} - %msg%n%rEx</pattern>
         </encoder>
     </appender>
 
+    <appender name="FILE_RESP" class="ch.qos.logback.core.FileAppender">
+        <file>target/response-${date}.log</file>
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%-5level] %logger{15} - %msg%n%rEx</pattern>
```

Comment:
Nit, do we need a time format that is so precise?  
It's been a while since I've coded in Java, but I'm guessing `ss.SSS` is really precise.
If there is a need for that much precision, then awesome.  
Also, what does `[%-5level] %logger{15} - %msg%n%rEx` resolve to? 
Also, what does the __15__ mean in `%logger{15}`.  Really just curious on that one.

Again, this is all just a nit, so no worries.

Reply:
I copied pattern from Gatling official repo, tbh I didn't pay attention to precision. I will have a look in logback docs to see what it is about.